src/plugins/data/public/ui/typeahead/_suggestion.scss

```
@@ -10,23 +10,31 @@ $kbnTypeaheadTypes: (
   max-height: 20vh;
 }
 
+.kbnTypeahead__popover--top {
+  box-shadow:
+    0 -2px 4px -1px transparentize($euiShadowColor, .8),
+    0 0 2px 0 transparentize($euiShadowColor, .8);
```

Thanks! A couple suggestions for the top positioned popover

```suggestion
  @include euiBottomShadowFlat;
  border-top-left-radius: $euiBorderRadius;
  border-top-right-radius: $euiBorderRadius;
```